server/common/common/urls.py

```diff
@@ -15,8 +15,18 @@
 """
 from django.contrib import admin
 from django.urls import include, path
+from django.conf.urls import url
+from django.conf import settings
+from django.views.static import serve
 
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('api/v1/',include('api.urls'))
 ]
+
+if settings.DEBUG:
```

Comment:
Where will this be used? Please comment above this if block.

Reply:
add comment in file
it is used for Uploaded image file shows on web page